README.md

```diff
@@ -145,4 +145,75 @@ Updates to any environment variables must be updated here along side the core en
   - ansible-playbook configure_webservers.yml -i controlled_nodes.ini
 - Ensure that there are no errors
 - navigate to any host ip address in a public web browser:
-  - http://managednodeip:5000
\ No newline at end of file
+  - http://managednodeip:5000
+
+
+
+
+## Deploying on Server(s) using Docker
+
+### Notes for Corporate Proxy Configuration
+
+Docker when running behind corporate proxy may require a proxy set within the Docker app itself to enable package connectivity and then proxy config setting to enable internal internet access to the docker image.
+
+Install local proxy as normal, opening localhost:3128
+
+Within Windows Docker, Add proxy settings:
+- Web Serer (HTTP) = http://localhost:3128
+- Web Server (HTTPS) = http://localhost:3128
+- BYpass proxy settings for these hosts & domains =  localhost,127.0.0.1,.CORPNAMEHERE.com
+
+Within Docker config file add proxy settings:
+C:\Users\<user>\.docker\config.json
+
+"proxies":
+ {
+   "default":
+   {
+     "httpProxy": "http://host.docker.internal:3128",
+     "httpsProxy": "http://host.docker.internal:3128",
+     "noProxy": "host.docker.internal,localhost,127.0.0.1,.CORPNAMEHERE.com"
+   }
+ }
+
+### Docker Commands
+
+#### Build
+
+Build the relevant environment (dev / prod) using the following commands:
+- docker build --target development --tag todo-app:dev .
+- docker build --target production --tag todo-app:prod .
+
+#### Run
+
+Run the docker project in the relevant environment (dev / prod) using the following commands:
+- docker run -ti --mount type=bind,source="$(pwd)/todo_app",target=/opt/todoapp/todo_app  -p 5000:5000 --env-file .env -d todo-app:dev
+- docker run -ti --mount type=bind,source="$(pwd)/todo_app",target=/opt/todoapp/todo_app  -p 5000:5000 --env-file .env -d todo-app:prod
```

Comment:
Not that it's a _bad_ thing, but I wouldn't expect a bind mount here, since we won't generally want to be hot reloading the code for our production instance!
```suggestion
- docker run -ti  -p 5000:5000 --env-file .env -d todo-app:prod
```